noggin/form/edit_user.py

```diff
@@ -76,6 +76,8 @@ class UserSettingsProfileForm(BaseForm):
         validators=[Optional()],
     )
 
+    pronouns = StringField(_('Pronouns'), validators=[Optional()],)
```

Comment:
If we provide the full flexibility of a string field here, I'm worried that we'll end up with vastly different values that are actually pretty much the same ("she", "she/her", "woman", etc...).
At first I think we should provide a select field with "she / her", "he / him", "they / them" and "other", to limit possibilities.
Then we could find a way for a free-form field to appear when "other" is selected. I've tried mocking that up but it's taking me too long (jquery...) so I think we can start with the simple select field. What do you think?

Reply:
updated to provide some default pronoun sets